code/sorting/src/cocktail_sort/cocktail_sort.py

```diff
@@ -0,0 +1,60 @@
+# cocsort is the function used to sort array
+# it is just like bubble sort with less complexity comparatively
+def cocsort(l):
```

Comment:
Rename this function to `cocktail_sort`. Make sure to update the comment above. Also, if you have not already, reformat this file using Black.